command.py

```diff
@@ -71,10 +71,11 @@
     'set_returnHomeCoordinates':{'cmd':130, 'type':'ddf'},
     'tare_IMU':                 {'cmd':131,  'type':'fff'},
     'set_IMU':                  {'cmd':132,  'type':'fff'},
-    'set_KDValues':              {'cmd':133,  'type':'fffffff'},
+    'set_KDValues':              {'cmd':133,  'type':'fffffff'}, 
```

Comment:
If you're using an IDE like pycharm, theres a shortcut to do this (Ctrl+Alt+L) so you don't have to do this manually every time